### ISSUE TITLE
Internal server error when using with subs-manager

### ISSUE THREAD

**Issue by lorensr** [Lepozepo/meteor-publish-with-relations#3]
I haven't yet noticed it affecting functionality.

```coffeescript
#client
home_subs = new SubsManager
  cacheLimit: 9999
  expireIn: 9999

home_subs.subscribe 'friends'

#server
Meteor.publish 'friends', ->
  Meteor.publishWithRelations ...
```

https://github.com/meteorhacks/subs-manager

![image](https://cloud.githubusercontent.com/assets/251288/7745363/6fe9ee9a-ff5f-11e4-8745-2c707f0c8768.png)

No error is logged server-side.

**Comment by lorensr** [comment]
Could be related: also can't publish multiple `publishWithRelations` in one publication:
```coffeescript
Meteor.publish 'friends', ->
  [
    Meteor.publishWithRelations ...
    Meteor.publishWithRelations ...
  ]
```
![image](https://cloud.githubusercontent.com/assets/251288/7746170/2a5aa170-ff65-11e4-947d-583a541d5a63.png)

**Comment by Lepozepo** [comment]
Haha, yeah, it does work with subs manager the problem is how the publisher is formed. If you want to use multiple ones you need to do something like this:


```coffeescript
Meteor.publish "friends", ->
  Meteor.publishWithRelations
    _noReady:true
  Meteor.publishWithRelations

  @ready()
```

It's not documented yet since it's the way the original package worked. I'll add it to the docs though.

**Comment by lorensr** [comment]
Your second call doesn't have the `_noReady` arg – in general, the first call to `publishWithRelations` should have the `_noReady:true`, or all the but the last call?

**Comment by Lepozepo** [comment]
Everything except the last unless you're returning `@ready()` at the end then you could have _noReady on all of them

**Comment by lorensr** [comment]
Got it. Multiple `.publishWithRelations` per `.publish` is working, however I still get the same original Internal server error when using with SubsManager.

**Comment by Lepozepo** [comment]
Weird, are you sure you're doing `home_subs.subscribe 'friends'` and not `SubsManager.subscribe 'friends'`?

**Comment by lorensr** [comment]
Yep! Not sure how to further diagnose, since the client-side error is
non-descriptive, and there's no error printed server-side – I guess
subs-manager must be catching the error?

**Comment by Lepozepo** [comment]
Weird, I'll test when I get a chance to see if it's on our side.
